.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+product.log
```

```suggestion
.log
```
this is better because it will add all .log files to .gitignore